build.gradle

```diff
@@ -23,12 +23,13 @@ repositories {
     maven { url = 'https://jitpack.io/' }
     maven { url = 'https://repo.minebench.de/' }
     maven { url = 'https://mvn.exceptionflug.de/repository/exceptionflug-public/' }
+    maven { url = "https://maven.elytrium.net/repo/" }
```

Comment:
Elytrium isn't needed here as Exceptionflug's repo already provides `velocity-proxy`. Up to you, but one of these two can probably be removed. Shame there's no more official source for depending on velo internals.